urdf/include/urdf/model.h

```diff
@@ -48,24 +49,39 @@
 namespace urdf
 {
 
+// PIMPL Forward Declaration
+class ModelImplementation;
+
+/// \brief Populates itself based on a robot descripton
+///
+/// This class uses `urdf_parser_plugin` to parse the given robot description.
+/// The picked plugin is the one that reports the most confident score.
```

Comment:
A nit to be sure: "The chosen plugin" or "The selected plugin" seems clearer to me than "The picked plugin".

Reply:
picked -> chosen in 70a81cc